src/Instances.js

```diff
@@ -92,61 +95,38 @@ class Instances extends React.Component {
-              to makeQueryFunction.
-              https://issues.folio.org/browse/STRIPES-480
-            */
-            const resourceData = args[2];
-            const sortMap = {
-              Title: 'title',
-              publishers: 'publication',
-              Contributors: 'contributors',
-            };
-
-            const index = resourceData.query.qindex ? resourceData.query.qindex : 'all';
-            const searchableIndex = searchableIndexes.find(idx => idx.value === index);
-
-            let makeQueryArgs = {};
-            if (index === 'isbn' || index === 'issn') {
-              const identifierType = resourceData.identifier_types.records.find(type => type.name.toLowerCase() === index);
-              makeQueryArgs = { identifierTypeId: (identifierType ? identifierType.id : 'identifier-type-not-found') };
-            }
-
-            let cql = searchableIndex.makeQuery(resourceData.query.query || '', makeQueryArgs);
-
-            const filterCql = filters2cql(filterConfig, resourceData.query.filters);
-            if (filterCql) {
-              if (cql) {
-                cql = `(${cql}) and ${filterCql}`;
-              } else {
-                cql = filterCql;
-              }
-            }
-
-            const { sort } = resourceData.query;
-            if (sort) {
-              const sortIndexes = sort.split(',').map((sort1) => {
-                let reverse = false;
-                if (sort1.startsWith('-')) {
-                  // eslint-disable-next-line no-param-reassign
-                  sort1 = sort1.substr(1);
-                  reverse = true;
-                }
-                let sortIndex = sortMap[sort1] || sort1;
-                if (reverse) {
-                  sortIndex = `${sortIndex.replace(' ', '/sort.descending ')}/sort.descending`;
-                }
-                return sortIndex;
-              });
-
-              cql += ` sortby ${sortIndexes.join(' ')}`;
+            const [
+              queryParams,
+              pathComponents,
+              resourceData,
+              logger
+            ] = args;
+            const queryIndex = resourceData.query.qindex ? resourceData.query.qindex : 'all';
+            const searchableIndex = searchableIndexes.find(idx => idx.value === queryIndex);
+            let queryTemplate = '';
+
+            if (queryIndex === 'isbn' || queryIndex === 'issn') {
+              const identifierType = resourceData.identifier_types.records.find(type => type.name.toLowerCase() === queryIndex);
+              const identifierTypeId = identifierType ? identifierType.id : 'identifier-type-not-found';
+              queryTemplate = searchableIndex.makeQuery(identifierTypeId);
+            } else {
+              queryTemplate = searchableIndex.makeQuery();
             }
 
-            return cql;
-          },
+            resourceData.query = { ...resourceData.query, ...{ qindex: '' } };
```

Comment:
What about 

> resourceData.query = { ...resourceData.query, qindex: '' };
?
As for me it looks easier.

Reply:
Thanks. Fixed